.vscode/launch.json

```diff
@@ -22,6 +22,19 @@
       "request": "launch",
       "name": "Launch Program",
       "program": "${file}"
+    },
```

Comment:
Revert before merging

Reply:
Fixed in 77ed0b21bb95afb86a610213fb4271ed7235aa9b